openmetadata-ui/src/main/resources/ui/src/components/EntityList/EntityList.tsx

```diff
@@ -90,14 +90,17 @@ export const EntityListWithAntd: FunctionComponent<AntdEntityListProp> = ({
   loading,
 }: AntdEntityListProp) => {
   return (
-    <Card
-      className="panel-shadow-color"
-      extra={headerText}
-      title={headerTextLabel}>
+    <Card className="panel-shadow-color">
       <EntityListSkeleton
         dataLength={entityList.length !== 0 ? entityList.length : 5}
         loading={Boolean(loading)}>
         <>
+          <Space className="w-full justify-between p-b-sm">
```

Comment:
Instead of giving custom classes to the `Space` component, can we use the `Row/Col` component s instead? This might lead to some overflow issues.

Reply:
Thanks!